src/deploy.sh

```diff
@@ -63,14 +64,16 @@ function kernel_deploy()
   reboot="${options_values['REBOOT']}"
   modules="${options_values['MODULES']}"
   single_line="${options_values['LS_LINE']}"
+  list_all="${options_values['LS_ALL']}"
   list="${options_values['LS']}"
   uninstall="${options_values['UNINSTALL']}"
+  uninstall_force="${options_values['UNINSTALL_FORCE']}"
   remote="${remote_parameters['REMOTE']}"
 
-  if [[ "$list" == 1 || "$single_line" == 1 ]]; then
+  if [[ "$list" == 1 || "$single_line" == 1 || "$list_all" == 1 ]]; then
     say "Available kernels:"
     start=$(date +%s)
-    list_installed_kernels "" "$single_line" "$target"
+    list_installed_kernels "" "$single_line" "$target" "$list_all"
```

Comment:
Since we are here, can we pass `$flag` to the first parameter?

Reply:
Isn't it already?

Reply:
Hi @joaoseckler ,

I think I did not explain really well what that I mean about using the flag here. Allow me to elaborate a little bit better... Basically, when I use your implementation, I can see the following output:
```
 sudo mkdir -p '/root/kw_deploy' 
 sudo touch '/root/kw_deploy/MANAGED_KERNELS'
```
I believe that we don't need to show that to users. Additionally, if we use the flag variable, we have more ways to test the command sequence. For trying to illustrate my change suggestion, take a look at the below diff:
```diff
diff --git a/src/deploy.sh b/src/deploy.sh
index ffc548d..9dd2915 100644
--- a/src/deploy.sh
+++ b/src/deploy.sh
@@ -47,6 +47,7 @@ function kernel_deploy()
   local end=0
   local runtime=0
   local ret=0
+  local flag
   local list_all
 
   if [[ "$1" =~ -h|--help ]]; then
@@ -69,11 +70,12 @@ function kernel_deploy()
   uninstall="${options_values['UNINSTALL']}"
   uninstall_force="${options_values['UNINSTALL_FORCE']}"
   remote="${remote_parameters['REMOTE']}"
+  flag=${options_values['test_mode']:-'SILENT'}
 
   if [[ "$list" == 1 || "$single_line" == 1 || "$list_all" == 1 ]]; then
     say "Available kernels:"
     start=$(date +%s)
-    list_installed_kernels "" "$single_line" "$target" "$list_all"
+    list_installed_kernels "$flag" "$single_line" "$target" "$list_all"
     end=$(date +%s)
 
     runtime=$((end - start))
@@ -309,7 +311,7 @@ function list_installed_kernels()
       list_installed_kernels '' "$single_line" '' "$all"
       ;;
     3) # REMOTE_TARGET
-      local cmd="bash $REMOTE_KW_DEPLOY/deploy.sh --list_kernels '' '$single_line' '' '$all'"
+      local cmd="bash $REMOTE_KW_DEPLOY/deploy.sh --list_kernels '$flag' '$single_line' '' '$all'"
       remote="${remote_parameters['REMOTE_IP']}"
       port="${remote_parameters['REMOTE_PORT']}"
```